Client/Api/Commands/ITenantIdsCommandStep.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections.Generic;
+
+namespace Zeebe.Client.Api.Commands
+{
+    public interface ITenantIdsCommandStep<out T>
+    {
+        /// <summary>
+        /// Set a list of tenantIds to associate with this resource.
+        /// </summary>
```

Comment:
```suggestion
        /// </summary>
        /// 
        /// This can be useful when requesting jobs for multiple tenants at once. Each of the activated 
        /// jobs will be owned by the tenant that owns the corresponding process instance.
```